community/intel-media-driver/build

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh -e
+
+export DESTDIR="$1"
+
+sed -i 's/SHARED/STATIC/' gmmlib/Source/GmmLib/CMakeLists.txt
+
+cd media
+
+cmake -B build \
+	-DCMAKE_INSTALL_PREFIX=/usr \
+	-DCMAKE_INSTALL_LIBDIR=/usr/lib \
+    -DBS_DIR_GMMLIB="$OLDPWD/gmmlib/Source/GmmLib" \
+	-DINSTALL_DRIVER_SYSCONF=OFF \
+    -DRUN_TEST_SUITE=OFF
```

Comment:
Indentation is still off (4 spaces).

Reply:
This is odd, on my branch it's just fine.
[SEE](https://github.com/mmatongo/community/blob/intel-media-driver/community/intel-media-driver/build)